source/ViewFileList.cpp

```diff
@@ -6,8 +6,8 @@
 #include "WindowPeek.h"
 
 
-ViewFileList::ViewFileList( BRect R, char* name, Setup* s, Language* w) 
-:  BListView( R, name , B_SINGLE_SELECTION_LIST, B_FOLLOW_ALL_SIDES, B_FRAME_EVENTS | B_WILL_DRAW)
+ViewFileList::ViewFileList(char* name, Setup* s, Language* w) 
+:  BListView(name , B_SINGLE_SELECTION_LIST, B_FRAME_EVENTS | B_WILL_DRAW | B_SUPPORTS_LAYOUT) //This supports Layout Management.
```

Comment:
You don't need to add B_SUPPORTS_LAYOUT to non-rect constructor calls because it is done for you:
https://git.haiku-os.org/haiku/tree/src/kits/interface/View.cpp#n425